### PR TITLE
Fix/13775 refactoring warn others logic

### PR DESCRIPTION
**Pull request by puneetmahali1**
## Description
Code refactoring for the Warn For Others Screen and it's related to the ticket👇
(EXPOSUREAPP-11116) - https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11116

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13775

## Screenshots
UQS
Warn Others

https://user-images.githubusercontent.com/72390277/184136502-42f25f63-df97-4a06-97c3-d102116d1fc2.MP4

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/QRScanner/QRScannerViewController.swift [hunk 1]
```diff
@@ -114,6 +114,14 @@ class QRScannerViewController: UIViewController {
 	private var previewLayer: AVCaptureVideoPreviewLayer! { didSet { updatePreviewMask() } }
 	private var viewModel: QRScannerViewModel?
 	private var presenter: QRScannerPresenter?
+	private var isPresenterOnBehalfFlow: Bool { presenter != nil && presenter == .onBehalfFlow }
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/QRScanner/QRScannerViewController.swift:117]:
```suggestion
	private var isOnBehalfFlow: Bool { presenter != nil && presenter == .onBehalfFlow }
```

**Comment by Ein-Tim** [comment]
For the future: Please also provide screenshots here on GitHub and not only to internal, not public tickets.

File: src/xcode/ENA/ENA/Source/Scenes/QRScanner/QRScannerViewController.swift (hunk 1 above)
Comment (puneetmahali1) [reply src/xcode/ENA/ENA/Source/Scenes/QRScanner/QRScannerViewController.swift]: Thanks Naveed. Done 👍

**Review by flxschmidt — APPROVED** [review]
:)

**Review by naveeddotio — APPROVED** [review]

**Pull request merged**